training/model_evaluation/src/evaluate_model.py

```diff
@@ -208,7 +223,7 @@ def test_model():
     parser.add_argument("--config-file-name", dest="config_file_name", required=False, type=str, default=None)
     parser.add_argument("--output", type=str, dest="output")
     parser.add_argument("--device", type=str, required=False, default="cpu", dest="device")
-    parser.add_argument("--batch-size", type=int, required=False, default=None, dest="batch_size")
+    parser.add_argument("--batch-size", type=int, required=False, default=1, dest="batch_size")
```

Comment:
any reason to keep default for batch size = 1. For LLMs, we had decided to keep it 'None' so as to adopt the batch size that hf pipeline internally sets.